spec/factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    email { 'test_user@example.com' }
```

Comment:
you can use the gem [faker](https://github.com/faker-ruby/faker) to create fake data